stdlib/Markdown/src/Julia/interp.jl

```diff
@@ -11,7 +11,7 @@ function interpinner(stream::IO, greedy = false)
     startswith(stream, '$') || return
     (eof(stream) || peek(stream, Char) in whitespace) && return
     try
-        return _parse(stream::IOBuffer, greedy = greedy)
```

Comment:
😭 

Can we also check the exception type?

Reply:
I don't quite know what exception type to look for, tests pass even if I comment out the try-catch so perhaps we can just do that?